deployment/jcdc.jsonnet

```diff
@@ -3,8 +3,8 @@
     hostname: null,
     docker_tag: 'latest',
   },
-  configure(hostname=null, docker_tag=null)::
-    self {
+  configure(overlay={}, hostname=null, docker_tag=null)::
+    self +overlay+{
```

Comment:
I don't think you ran jsonnetfmt on this - it should have added spaces around the `+`s I think